### ISSUE TITLE
automated reloading of files

### ISSUE THREAD

**Issue by fotofiete** [BoostIO/Boostnote#3277]
Hi!
We are using boostnote in our team and it works like a charm! We put the files into a shared folder (synchronized via seafile) so that everybody can see the projects notes. The only problem is, that boostnote needs a restart in order to recognize changes by other team members.
It would be a great feature if boostnote automatically reloads notes when changed in the storage location! This has been discussed in #521 but that ticket has been closed last year...

**Comment by Flexo013** [comment]
Agreed that this is in essence a step towards fully implementing #308
